native/src/modules/app/constants/buildConfig.js

```diff
@@ -3,12 +3,14 @@
 import type { CommonBuildConfigType } from 'build-configs/BuildConfigType'
 import loadBuildConfig, { COMMON } from 'build-configs'
 import integreatAppIcon from 'build-configs/integreat/assets/app-icon-round.png'
+import integreatLoadingImage from 'build-configs/integreat/assets/app-logo-inverted.svg'
 import integreatLocationMarker from 'build-configs/integreat/assets/location-marker.svg'
 import integreatIntroLanguageIcon from 'build-configs/integreat/assets/intro-slides/Language.svg'
 import integreatIntroEventsIcon from 'build-configs/integreat/assets/intro-slides/Events.svg'
 import integreatIntroOffersIcon from 'build-configs/integreat/assets/intro-slides/Offers.svg'
 import integreatIntroSearchIcon from 'build-configs/integreat/assets/intro-slides/Search.svg'
 import malteAppIcon from 'build-configs/malte/assets/app-icon-round.svg'
+import malteLoadingImage from 'build-configs/malte/assets/app_icon_malte_round.png'
```

Comment:
```suggestion
import malteLoadingImage from 'build-configs/malte/assets/app-icon-circle.png'
```
Otherwise you should change `app-icon-round.svg` to `app-icon-rounded.svg` I think to avoid confusion.

Reply:
I used the image from main/res/mipmap. If I rename it the connection gets lost. But I think that isn't a big problem, I will rename it